hikari/impl/rest.py

```diff
@@ -3512,24 +3547,34 @@ async def edit_interaction_response(
         ] = undefined.UNDEFINED,
     ) -> messages_.Message:
         route = routes.PATCH_INTERACTION_RESPONSE.compile(webhook=application, token=token)
-        return await self._edit_message(
-            route,
-            data_binding.JSONObjectBuilder(),
-            no_auth=True,
+
```

Comment:
Some of this spacing seems a bit unnecessary (although the spaces around the `if/if else` statement is fine)